assets/templates/partials/release/contents/more-on-this-topic.tmpl

```diff
@@ -0,0 +1,20 @@
+<div class="more-on-this-topic">
+    <ol class="ons-list ons-list--bare ons-u-bt ons-u-mb-l">
+        {{ range .RelatedAPIDatasets }}
+            <li class="ons-list__item ons-u-mt-l">
+                {{ if and .URI .Title }}
+                  <a href="{{ .URI }}">{{ .Title }}</a>
+                {{ else }}
```

Comment:
This `else` assumes that `.URI` and `.Title` are both un-populated but doesn't cover that the `.URI` could actually be empty too. It should be something like...
```suggestion
                {{ else if .URI }}
```